roles/splunk_cluster_master/tasks/configure_indexes.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Add stanza for the index specified
+  ini_file:
+    dest: "{{ splunk.home }}/etc/master-apps/_cluster/local/indexes.conf"
+    section: "{{ index.index }}"
```

Comment:
NIT but this seems a bit confusing. Maybe `index.name` or `index.indexName` would be more explicit?

Reply:
Fixed.